### PR TITLE
Chapter 11: Create simple UI

### PR DESCRIPTION
**Pull request by tungbd-0214**

### CODE REVIEW

**Comment by tungbd-0214** [comment]
ready

File: routes/web.php [hunk 1]
```diff
@@ -15,24 +16,24 @@
...
+    Route::get('/users/{user}/edit', [UserController::class, 'edit'])
+                ->name('users.edit');
+    Route::get('/users/{user}', [UserController::class, 'show'])
+                ->name('users.show');
+    Route::post('/users', [UserController::class, 'store'])
+                ->name('users.store');
+    Route::put('/users/{user}', [UserController::class, 'update'])
+                ->name('users.update');
+    Route::delete('/users/{user}', [UserController::class, 'delete'])
+                ->name('users.delete');
```
Comment (ghost) [inline routes/web.php]: indent 4 spaces

**Comment by tungbd-0214** [comment]
ready

**Comment by ghost** [comment]
merged

**Pull request merged**